README.md

```diff
@@ -469,6 +476,48 @@ local commands = {
 <h3>
 <!-- panvimdoc-ignore-end -->
 
+Functions
+
+<!-- panvimdoc-ignore-start -->
+
+</h3>
+</summary>
+<!-- panvimdoc-ignore-end -->
+
+Functions follow a similar structure as anonymous commands, but description is **required**. If called in visual mode, the visual selection is passed as an argument of the form `{ cline, ccol, vline, vcol }`, where `c` corresponds to the cursor position, and `v` the visual selection (see `:h line()` and `:h col()`).
```

Comment:
> If called in visual mode, the visual selection is passed as an argument of the form `{ cline, ccol, vline, vcol }`, where `c` corresponds to the cursor position, and `v` the visual selection (see `:h line()` and `:h col()`).

Can you update this bit to use your description for consistency? I think your description is a bit more clear than what I've written here. https://github.com/mrjones2014/legendary.nvim/blob/master/README.md?plain=1#L373

I think this is good to go once we do that.